app/controllers/api/v1/users_controller.rb

```diff
@@ -14,9 +14,9 @@ def show
   def index
     if params[:user][:search]
       role = if current_user.shop?
-        2
+        "Shipper"
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Metrics/AbcSize
**Severity**: INFO
**File**: app/controllers/api/v1/users_controller.rb L17

Assignment Branch Condition size for index is too high. [26.65/15]

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*